net-vpn/zerotier/zerotier-1.2.4.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+inherit flag-o-matic systemd
+
+DESCRIPTION="A smart ethernet switch for planet Earth"
+HOMEPAGE="https://www.zerotier.com"
+SRC_URI="https://github.com/${PN}/ZeroTierOne/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~x86-linux ~amd64-linux ~arm ~mips"
```

Comment:
I talked to @leio and we don't want `~mips` keywords on a package unless you have explicit request from users who are actually going to use this package on a MIPS system. I suggest following a similar rule for other minor platforms.

The point is that keywording usually involves future maintenance work. We have a major issue like that with many Python packages — someone added support for some minor platform back in the day, and now we have to deal with requesting keywording on a lot of additional dependencies, with nobody actually using the package and arch teams not having resources to test it all.